tools/modules/EnsEMBL/Web/Component/Tools/VR/Results.pm

```diff
@@ -319,6 +360,110 @@ sub linkify {
   return @return_values;
 }
 
+sub linkify_mane {
+  my $self = shift;
+  my $field = shift;
+  my $values = shift;
+  my $species = shift;
+  my $job_data = shift;
+  my $mane_count_g = shift;
+
+  # work out core DB type
+  my $db_type = 'core';
+  if(my $ct = $job_data->{core_type}) {
+    if($ct eq 'refseq' || ($values && $ct eq 'merged' && $values !~ /^ENS/)) {
+      $db_type = 'otherfeatures';
+    }
+  }
+
+  my @return_values = ();
+  my %return_values_hgvsg;
+  my $new_value;
+  my $hub = $self->hub;
+  my $sd = $hub->species_defs;
+
+  my @all_values = split(', ', $values);
+
+  foreach my $value (@all_values) {
+    return '-' unless defined $value && $value ne '';
+
+    my $mane_result;
+    my ($hgvsg, $hgvsc, $hgvsp) = split /\;/, $value;
+
+    if($hgvsc =~ /^ENST/) {
+      my @split_value = split(':', $hgvsc);
+      my $url = $hub->url({
+        type    => 'Transcript',
+        action  => 'Summary',
+        t       => $split_value[0],
+        species => $species,
+        db      => $db_type,
+      });
+
+      my $zmenu_url = $hub->url({
+        type    => 'ZMenu',
+        action  => 'Transcript',
+        t       => $split_value[0],
+        species => $species,
+        db      => $db_type,
+      });
+
+      my $transc_value = $self->zmenu_link($url, $zmenu_url, $split_value[0]);
+      $mane_result = $transc_value.":".$split_value[1];
+    }
+    else {
+      # RefSeq transcripts
+      $mane_result = $hgvsc;
+    }
+
+    if(defined($hgvsp) && $hgvsp =~ /^ENSP/) {
+      my @split_value = split(':', $hgvsp);
+      my $url = $hub->url({
+        type    => 'Transcript',
+        action  => 'ProteinSummary',
+        t       => $split_value[0],
+        species => $species,
+        db      => $db_type,
+      });
+
+      my $zmenu_url = $hub->url({
+        type    => 'ZMenu',
+        action  => 'Transcript',
+        t       => $split_value[0],
+        species => $species,
+        db      => $db_type,
+      });
+
+      my $transp_value = $self->zmenu_link($url, $zmenu_url, $split_value[0]);
+      $mane_result .= "&nbsp;(".$transp_value.":".$split_value[1].")";
```

Comment:
I don't like the design of adding the protein level description as an addition in brackets, it looks a little forced. Probably worth (another, sorry!) a team-wide discussion about exactly how we want this to look.

Reply:
Adding brackets was the only way I found to show the HGVS protein without being too much but it's not the best display. We can have a chat about the display on the next team meeting.